packages/web-react/src/types/pricingPlan.ts

```diff
@@ -0,0 +1,50 @@
+import { ComponentPropsWithRef, ElementType, JSXElementConstructor, ReactNode } from 'react';
+import { ChildrenProps, StyleProps } from './shared';
+
+export interface PricingPlanBaseProps extends ChildrenProps, StyleProps {
+  /** If pricing plan is comparable  */
+  isComparable?: boolean;
```

Comment:
As per @crishpeen's request to be more accurate regarding this feature, we should change the name to `hasComparableFeatures`:

```suggestion
  hasComparableFeatures?: boolean;
```

Do you agree?